cfme/configure/tasks.py

```diff
@@ -36,12 +39,17 @@
     ]
 )
 
+table_loc = '//div[@id="records_div"]/table'
```

Comment:
Tested, tried, but I wasn't able to properly reference it from within the nested views.